src/ihela_client/merchant_client.py

```
@@ -95,13 +96,17 @@ def authenticate(self):
             # TODO : Delete this line for production
             logger.debug(auth_data)
 
-        auth_ = requests.post(self.get_url(url), auth=(self.client_id, self.client_secret), data=auth_data)
+        auth_ = requests.post(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)